### PR TITLE
feat(material/chips): add unit tests for custom aria-describedby

### PR DESCRIPTION
**Pull request by RobertAKARobin**
Continuation of #24292

See also #24644 for MatSelect

### CODE REVIEW

**Comment by devversion** [comment]
@RobertAKARobin M1 should work to my knowledge. We should close the issue, unless you have more details.

**Comment by RobertAKARobin** [comment]
@devversion I'll follow up in that other issue. Once I've got `yarn test mdc-chips` working I'd be happy to update this.

**Comment by RobertAKARobin** [comment]
@devversion Updated! The `mat-chips` specs test its usage in `mat-form-field`, but there aren't any existing FormField specs for `mdc-chips`. Adding all that felt like it was outside the scope of this PR.

File: src/material-experimental/mdc-chips/chip-set.ts [hunk 1]
```diff
@@ -85,6 +86,17 @@ export class MatChipSet
...
+  @Input('aria-describedby')
+  get userAriaDescribedBy(): string {
+    return this._userAriaDescribedBy;
+  }
+
+  set userAriaDescribedBy(userAriaDescribedBy: string) {
+    this._userAriaDescribedBy = userAriaDescribedBy;
+  }
+
+  private _userAriaDescribedBy: string;
```
[the remark this answers is not in the document]
Comment (RobertAKARobin) [reply src/material-experimental/mdc-chips/chip-set.ts:98]: @andrewseguin Can you explain a bit more about what changes you were looking for?

[the remark this answers is not in the document]
Comment (RobertAKARobin) [reply src/material-experimental/mdc-chips/chip-set.ts:98]: @andrewseguin Can you explain a bit more about what changes you were looking for?

@devversion Yes, the getters/setters seem unnecessary. I was copying from the mat-chips code (which in turn was copied from mat-select, IIRC). Will wait to see what Andrew says though.

**Comment by andrewseguin** [comment]
Closing since this change is a bit out of date, especially since we've removed the legacy chips and shuffled code around. Feel free to re-open with a rebased set of updates!

**Pull request closed without merging**